File: Cloth-Store-Warehouse-BE/ct-core-model/src/main/java/com/example/ctcoremodel/AgencyModel.java
package com.example.ctcoremodel;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.time.LocalDateTime;

@NoArgsConstructor
@Data
@AllArgsConstructor
public class AgencyModel {
    private String id;
    private String name;
    private LocalDateTime createdDate;
    private LocalDateTime updatedDate;
    private String phone;
    private String address;
    private String code;
    private String companyId;
}

File: Cloth-Store-Admin-FE/src/app/core/apis/Dtos/Agency.model.ts
import {DatePipe} from "@angular/common";

export class AgencyModel {
  id!: string | null;
  name!: string | null;
  createdDate!: DatePipe | null;
  updatedDate!: DatePipe | null;
  phone!: string | null;
  address!: string | null;
  code!: string | null;
  companyId!: string | null;
}
